src/ctim/schemas/malware.cljc

```diff
@@ -21,17 +22,24 @@
 (def malware-desc-link
   "[Malware](https://docs.google.com/document/d/1IvkLxg_tCnICsatu2lyxKmWmh1gY2h8HUNssKIE-UIA/pub#h.s5l7katgbp09)")
 
+(def-entity-type MalwareName
+  (str "A name given by an anti-virus vendor to identify a malware sample.")
+  (f/required-entries
+   (f/entry :vendor c/ShortString)
+   (f/entry :name c/ShortString))
+  (f/optional-entries
+   (f/entry :signature-name c/ShortString)))
```

Comment:
What is the difference between the `name` and the `signature-name` field?

Reply:
example:
```
:names [{:vendor "Microsoft"
         :name "Gupboot.B"
         :signature_name "Trojan:Win32/Gupboot.B"}]
```

Reply:
Updated again to add descriptions for the required `:name` and optional `:signature_name` to help explain and distinguish them.